### PR TITLE
SAK-43474 TurnitinOC: Localization for Sakai integration

### PR DESCRIPTION
**Pull request by baholladay**

### CODE REVIEW

File: content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties [hunk 1]
```diff
@@ -35,9 +35,9 @@ report.error.unknown=Unknown error occurred retrieving an originality report
 
 submission.terminal.status.UNSUPPORTED_FILETYPE=The uploaded filetype is not supported
 submission.terminal.status.PROCESSING_ERROR=An unspecified error occurred while processing the submissions
-submission.terminal.status.TOO_LITTLE_TEXT=The submission does not have enough text to generate a Similarity Report (a submission must contain at least 20 words)
```
Comment (fostersdesign) [inline content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: Why are the useful informational bits being removed?

Comment (baholladay) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: These details are too specific. Each fact has already changed and is incorrect. It seems odd to have to maintain these in a properties file

Comment (fostersdesign) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: So these limits are no longer the case?

Comment (baholladay) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: sorry, looks like only one has changed. Too little text is the same, too many pages is 800 now, too much text is still 2MB

Comment (baholladay) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: (this is for TurnitinOC)

Comment (fostersdesign) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: Thanks for clarifying, @baholladay. Would it not be useful to still communicate this with end-users through these messages?

Comment (baholladay) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: I'll re-add the specific wording... I'll have to re-submit and retranslate so it'll be a few

Comment (fostersdesign) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: Thanks, @baholladay!

Comment (baholladay) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: updated! please review and merge

Comment (baholladay) [reply content-review/impl/turnitin-oc/src/main/resources/turnitin-oc.properties:38]: actually deleting this PR and opened a new one: https://github.com/sakaiproject/sakai/pull/8228

**Pull request closed without merging**